src/test/java/com/wikia/webdriver/common/core/urlbuilder/UrlBuilder.java

```diff
@@ -10,6 +10,11 @@
  */
 public class UrlBuilder {
 
+  final String XIPIO_ADDRESS_FORMAT = ".%s:%d";
+
```

Comment:
Why we need to provide an environment to UrlBuilder()? Shouldn't we take this data from configuration? How about  actually using a `Builder` pattern. En example usage in my head looks like this:
```
UrlBuilder.getUrlForWiki("MyCoolWikia").forEnv("preview").forBrowser("CHROME").build();
```
and a default one should use a browser and environment provided from ConfigurationFactory.class, so nothing changes in a regular usage:
```
UrlBuilder.getUrlForWiki("MyCoolWikia").build();
```